ops/terraform/env/mgmt/github-actions-iam.tf

```diff
@@ -421,7 +421,8 @@ resource "aws_iam_policy" "github_actions_ci_ops" {
           Action = [
             "iam:Get*",
             "iam:List*",
-            "iam:DeletePolicyVersion"
+            "iam:DeletePolicyVersion",
+            "iam:DeletePolicy"
```

Comment:
Indeed, this is extending the permissions to now allow for deletion of policies–because it's unbound/unrestricted, this allows for destroying any policy which is an issue in and of itself. However, the previous issue today on the role being underprivileged is a _little_ more involved. It's somewhat of a circular dependency and must be bootstrapped out-of-band or by another role all together. This should probably be left out and the PR closed.

Reply:
I can reverse the permission change now that changing the policy description appeared to be the problem.